Procfile

```diff
@@ -1 +1 @@
-worker: node src/spawn_tests.js
+worker: npm run compile && node lib/main.js test
```

Comment:
I guess I forgot to remove it. We don't need to pass it command line arguments anymore